py/desispec/workflow/desi_proc_funcs.py

```diff
@@ -350,26 +366,19 @@ def determine_resources(ncameras, jobdesc, queue, nexps=1, forced_runtime=None,
         #- new version using MPI on one node
         ncores, runtime = ncameras, (8+2*nexps) #ncameras, 10
     elif jobdesc == 'POSTSTDSTAR':
-        ncores, runtime = ncameras, 10
+        runtime = 10
+        if system_name[0:10] == 'perlmutter':
+            ncores = config['cores_per_node']
+        else:
+            ncores = ncameras
```

Comment:
In other "if perlmutter" blocks, you are throttling `ncores = config['cores_per_node']` which is less than it otherwise would have requested in order to keep the job on a single Perlmutter node.

In this case for POSTDSTAR, `config['cores_per_node']` is considerably more than otherwise would have been requested (ncameras=30) on other systems.  Is that purposeful and demonstrated to be beneficial?  e.g. most (all?) of the poststdstar code isn't parallelized beyond the camera-level, so if we're only going to be using 30 ranks then we shouldn't bother doing slow python imports on 128 ranks.  Do we need a "if perlmutter" special case here?

Reply:
@sbailey thanks for catching this. We do not need the special case so I removed it.